.github/workflows/automate-team-review-assignment.yml

```diff
@@ -0,0 +1,83 @@
+name: Add community label, Assign reviewers
+
+on:
+    pull_request_target:
+        types: [opened, ready_for_review]
+    issues:
+        types: [opened]
+
+concurrency:
+    group: automate-team-review-assignment-${{ github.event_name }}-${{ github.event.pull_request.number || github.event.issue.number }}
+    cancel-in-progress: true
+
+permissions:
+    contents: read
+    pull-requests: write
+    issues: write
+
+jobs:
+    add-community-label:
+        name: Add community label
+        if: ${{ github.event.action == 'opened' }}
+        runs-on: ubuntu-latest
+        steps:
+            - name: Check if community contribution
+              id: check
+              uses: actions/github-script@v7
+              with:
+                  script: |
+                      const username = '${{ github.event.pull_request.user.login || github.event.issue.user.login }}';
```

Comment:
The logic extracted from `.github/workflows/scripts/is-community-contributor.js`, which is used in another workflow - but dropping it is for another iteration of workflow consolidation.